### DISCUSSION TITLE
Gledopto GL-SD-001 (Binding Capabilities Clarification)

### DISCUSSION THREAD

**Discussion by roderik-maker** [Koenkk/zigbee2mqtt#22146] [Devices]
The device page for the Gledopto GL-SD-001 states that ["The device allows zigbee binding"](https://www.zigbee2mqtt.io/devices/GL-SD-001.html). Based on [the manual](https://manuals.plus/gledopto/gl-sd-001-zigbee-triac-ac-dimmer-manual#axzz8WsjpQ9v9) it would seem that the device supports remotes binding to the dimmer but it is unclear if you can also bind the dimmer to other Zigbee devices such as light bulbs. 

It would be great (although I appreciate it's a long shot) if someone who has the GL-SD-001 (or maybe a similar device) could check if it can also be used as a controller when binding to other devices. 

_This is a selfish request as I am looking for a mains powered device that can be built into an existing light switch to control (Tradfri) bulbs. Nevertheless I think others might also benefit from improving the documentation for the device so that it clarifies what binding (types?) are supported._

**Comment by jbitsolutions** [comment]
I'm wondering the same thing. We're you able to figure it out?

**Comment by RoGu777** [comment]
I've bought two GL-SD-301P dimmers, as they are also able to support binding according to[ it's Z2M page ](https://www.zigbee2mqtt.io/devices/GL-SD-301P.html). I tried to bind both with each other, so that they will have the same brightness, but in Z2M it is not possible.

If someone has an idea how to make Gledopto dimmers to bind - you are welcome to share!
